apps/extension/client/src/extension.ts

```diff
@@ -63,20 +64,22 @@ export async function activate(context: vscode.ExtensionContext) {
     "xstateLanguageServer",
     "XState",
     serverOptions,
-    clientOptions,
+    clientOptions
   );
 
   client.start();
 
   await client.onReady();
 
+  checkTypegenNestingConfiguration();

```

Comment:
I assume that this will also get executed right after the installation as this is called in the `activate` handler, right?